index.html

```diff
@@ -0,0 +1,267 @@
+          The sap is collected in large, typically stainless steel, tanks. 
+          This process requires precise timing and careful handling to ensure the sap’s freshness and quality, which are vital for producing high-quality syrup.
+        </p>
+      </article>
+
+      <article class="slide" id="image-slide-process-2", data-image-src="assets/boiling.jpg">
+        <h1>The Art of Boiling and Evaporating</h1>
+        <img src="assets/farm.svg" alt="icon" class="process-icon">
+        <img src="assets/falling.gif" alt="Icon" class="giffy-icon", id="falling">
+        <p>Once collected, the sap undergoes a transformation in the sugar house, where it is first filtered to remove any debris or impurities. 
+          The next step is <span style="color: #a92d04; letter-spacing: 3px; font-size: 18px; font-weight: bold; font-family: Roboto">BOILING</span>, where the sap is heated in a large evaporator, a specialized piece of equipment designed to efficiently evaporate water from the sap. 
+          The sap, which is about 98% water, is reduced to a thicker consistency. As the water evaporates, the sap gradually becomes more concentrated and sweeter. 
+          This process requires careful monitoring to avoid burning the syrup or boiling it too long. 
+          Once the sap reaches the appropriate sugar concentration, it is transferred to a finishing pan where it is brought to the final temperature and consistency, typically around 66% sugar. 
+          This stage requires skill to ensure the syrup’s flavor and texture are just right.
+        </p>
+      </article>
+
+      <article class="slide" id="image-slide-process-3", data-image-src="assets/filtering.jpeg">
+        <h1>The Sweet Finale of Grading and Bottling</h1>
+        <img src="assets/farm.svg" alt="icon" class="process-icon">
+        <img src="assets/packaging.gif" alt="Icon" class="giffy-icon", id="giffy-icon">
+        <p>The final steps in syrup production involve filtering, grading, and packing. 
+          After boiling, the syrup is filtered to remove any remaining impurities or sugar crystals that have formed during the boiling process. 
+          This ensures a smooth and clear product. <span style="color: #a92d04; letter-spacing: 3px; font-size: 18px; font-weight: bold; font-family: Roboto">GRADING</span> follows, where the syrup is assessed for color and flavor to determine its grade, which ranges from light to dark amber. 
+          The grading process is crucial as it affects the syrup’s market value and consumer preference. 
+          Once graded, the syrup is packaged into bottles or jars, often under sterile conditions to prevent contamination. 
+          The finished syrup is then labeled and prepared for distribution, ready to bring a touch of nature’s sweetness to the table.
+        </p>
+      </article>
+
+      <article class="slide" id="chart-image-slide", data-image-src="assets/maple-forest.jpg">
+        <select id="select-button"></select>
```

Comment:
I actually didn't know this was a `select` when I saw it. It _looks_ nice, but I didn't realize I could interact with it until I did a scan of the page with _axe DevTools_.

**suggestion:** It might be useful to add some sort of label that indicates that the user can choose an option. Otherwise the user would have to be lucky enough to move their mouse over the element.

Axe raised the following issue, which is very related to my point above:

> **[Select element must have an accessible name](https://dequeuniversity.com/rules/axe/4.10/select-name?application=AxeChrome)**
> 
> To solve this problem, you need to fix at least (1) of the following:
> - Form element does not have an implicit (wrapped) `<label>`
> - Form element does not have an explicit `<label>`
> - aria-label attribute does not exist or is empty
> - `aria-labelledby` attribute does not exist, references elements that do not exist or references elements that are empty
> - Element has no `title` attribute
> - Element's default semantics were not overridden with `role="none"` or `role="presentation"`